programs/fileio.c

```diff
@@ -513,12 +513,24 @@ static FILE* FIO_openSrcFile(const char* srcFileName)
         return NULL;
     }
 
+#ifdef _MSC_VER
+
+    if (!UTIL_isRegularFile(srcFileName)) {
```

Comment:
I don't love duplicating the same test with the same code.

I'd much rather have a single `UTIL_isRegularFile()` check on both platforms, and then an addition `UTIL_isFIFO()` check on non-win platforms.